src/main/kotlin/racingcar/GameManager.kt

```diff
@@ -0,0 +1,42 @@
+package racingcar
+
+object GameManager {
+    private val validation = Validation()
+    private lateinit var carNames: List<String>
+    private var attemptTimes: Int = 0
+    private lateinit var round: Round
+
+    fun generateCars(input: String) {
+        validation.validateInputHasComma(input)
+        carNames = validation.validateCarNames(input.split(",").map { it.trim() })
+        initializeRound()
+    }
+
+    private fun initializeRound() {
+        round = Round(carNames)
+    }
+
+    fun setAttemptTimes(input: String) {
+        attemptTimes = validation.getAttemptTimes(input)
+    }
+
+    fun getRound(): Round {
+        return round
+    }
+
+    fun playRound() {
+        repeat(attemptTimes) {
+            playOneRound()
+        }
+    }
```

Comment:
입출력 요구사항 실행 결과 예시의 안내 메시지 출력이 누락된 것 같습니다.

```suggestion
    fun playRound() {
        println("\n실행 결과")
        repeat(attemptTimes) {
            playOneRound()
        }
    }
```

Reply:
아예 실행결과 부분을 생각지도 못했어서 놓쳤던 부분인것 같습니다! 감사합니다 👍